portal-api/src/v1/users/validation/rules/select-at-least-one-bank.js

```diff
@@ -0,0 +1,37 @@
+const { ADMIN } = require('../../../roles/roles');
+
+const CONSTANT = require('../../../../constants');
+/**
+ * Validates that if the bank is present it is not empty
+ * @param {Object} user the existing user
+ * @param {Object} change the changes to make
+ * @returns {Array} either an empty array or an array containing an error object
+ */
+const selectAtLeastOneBank = (user, change) => {
+  if (!change.bank || change.bank.length === 0 || change.bank[0] === null) {
```

Comment:
Condition is still not simplified?

Reply:
If I simplify to `if (!change.bank || !change.bank.length)`  I see other functionality breaks. Please suggest any simplified condition.
![image](https://github.com/UK-Export-Finance/dtfs2/assets/113366256/13aa0ddc-3d94-4936-a945-bf876aff5e1b)